### ISSUE TITLE
Raw Command Has more then 1 space between some commands.

### ISSUE THREAD

**Issue by Wontell** [cdgriffith/FastFlix#318]
Hi,
This is low importance bug since it looks like it works anyway. Maybe its not even a bug.

But when I look in the raw command window i see that there are more then one space between some commands

Here is a copy paste from nvenc:
-m has 4 spaces
--audio-copy has 2 spaces

--seek 2100.0 --video-metadata clear -c hevc --vbr 30000 --bref-mode middle --preset default --tier high --lookahead 32 --aq --aq-strength 0 --colormatrix smpte170m --transfer smpte170m --colorprim smpte170m --output-depth 10 --multipass 2pass-full --mv-precision Auto --chromaloc auto --colorrange auto --avsync forcecfr  --audio-copy 1    -m default_mode:infer_no_subs -o 

here copy paste from ffmpeg:
-i has  5 spaces
-max_muxing_queue_size has 3 spaces
-c:v has 3 spaces
-tune:v has 5 spaces
-an  has 3 spaces
-pass has 3 spaces

ffmpeg.exe" -y -ss 2100.0     -i "C:\Encode\Dying Light 2 Stay Human 2022.03.06 - 04.50.25.01-CFR.mkv"   -max_muxing_queue_size 1024 -map 0:0 -vsync cfr  -c:v hevc_nvenc -pix_fmt p010le     -tune:v hq -color_primaries smpte170m -color_trc smpte170m -colorspace smpte170m -spatial_aq:v 0 -tier:v main -rc-lookahead:v 0 -gpu -1 -b_ref_mode disabled -profile:v main   -pass 1 -passlogfile "pass_log_file_3170f538458588398560" -b:v 6000k -preset:v slow -2pass 1  -an -sn -dn -f mp4 NUL

**Comment by cdgriffith** [comment]
This is mostly an "I'm lazy" thing, as it doesn't harm anything, and makes generating the commands a lot easier and looks better in code. 

For example if I have a list of three things that may or may not exist:

```python
my_list  = [       
"ffmpeg",
("--vbaq" if settings.vbaq else ""),
("--pe" if settings.pre_encode else ""),
("--pa" if settings.pre_analysis else ""),
"my_video.mp4"
]
```

Then I say "hey, make that all into a single string, with a space bettween each list item" (`" ".join(my_list)`) 

I would have something like:

```
ffmpeg    my_video.mp4
```

I used to have a follow up cleanup command that would remove duplicate spaces. Then came across instances where people had double spaces in their filenames, and some other issue I am blanking on. 

Totally a valid issue and would be nice to cleanup again in future, just investing energy in other places for the time being. Please leave the issue open so we can track it, and thank you opening it!

tl;dr; It shouldn't hurt anything, makes code easy to read and write as is, and don't have time at moment to invest in that cleanup.

**Comment by Wontell** [comment]
> tl;dr; It shouldn't hurt anything, makes code easy to read and write as is, and don't have time at moment to invest in that cleanup.

Hi,
Thank you for replay.
I reported just in case, consider this closed.

Regards
